modules/apps/bean-portlet/bean-portlet-cdi-extension-sample/src/main/java/com/liferay/bean/portlet/cdi/extension/sample/portlet/HelloWorldFacesPortlet.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.bean.portlet.cdi.extension.sample.portlet;
+
+import javax.portlet.PortletConfig;
+import javax.portlet.PortletException;
+import javax.portlet.faces.GenericFacesPortlet;
+
+/**
+ * @author  Kevin Valencia
+ */
+public class HelloWorldFacesPortlet extends GenericFacesPortlet {
+
+	@Override
+	public void init(PortletConfig portletConfig) throws PortletException {
+		Thread currentThread = Thread.currentThread();
+
+		ClassLoader classLoader = currentThread.getContextClassLoader();
+
+		System.err.println(" classLoader=" + classLoader);
+
+		super.init(portletConfig);
+	}
+
+}
```

Comment:
@Kvale1733, the override method is unnecessay. Please remove this method. Only keep
```
public class HelloWorldFacesPortlet extends GenericFacesPortlet {
}
```